Sources/Xcodeproj/pbxproj().swift

```diff
@@ -94,7 +94,7 @@ public func pbxproj(srcroot: String, projectRoot: String, xcodeprojPath: String,
         print("            buildPhases = (\(module.compilePhaseReference), \(module.linkPhaseReference));")
         print("            buildRules = ();")
         print("            dependencies = (\(module.nativeTargetDependencies));")
-        print("            name = \(module.name);")
+        print("            name = \(module.c99name);")
```

Comment:
SwiftPM is getting it wrong, then. We should match Xcode in our computation of module names... Xcode substitutes invalid characters with `_`.